test/multistream.test.js

```diff
@@ -503,21 +503,17 @@ test('flushSync', function (t) {
     '_' + Math.random().toString(36).substr(2, 9)
   )
   const destination = pino.destination({ dest: tmp, sync: false, minLength: 4096 })
-  const log = pino({ level: 'info' }, multistream([{ level: 'info', stream: destination }]))
+  const stream = multistream([{ level: 'info', stream: destination }])
```

Comment:
pino v8 side note: can we merge multistream and transport APIs in a way where the multistream same-thread logic is used based on available threads (or a force option)

Reply:
I suggest to park this till v9, it's better to ship what we have and improve in the next iteration.